modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -586,10 +586,25 @@ public function doCurlCall($url, $post_content='') {
 		return $responseObj;
 	}
 
+	public function getLangFilePath($language=false)	{
```

Comment:
Although calling non static methods is possible it will raise a warning an may deprecate in future. Add static

Reply:
Fixed in ec43d69